src/OrleansSQLUtils/Storage/RelationalStorage.cs

```diff
@@ -146,11 +146,7 @@ public Task<IEnumerable<TResult>> ReadAsync<TResult>(string query, Action<IDbCom
                 throw new ArgumentNullException("selector");
```

Comment:
@ReubenBond, @jdom The duplicate IDs could also be something like the pseudorandom generated being seeded again with the same seed (if some static was removed). In any event, it looks like there shouldn't be a logic failure. I might do some checking on this in a few hours.